server/mediaserver.go

```diff
@@ -701,6 +701,7 @@ func (s *LivepeerServer) HandlePush(w http.ResponseWriter, r *http.Request) {
 		hdrs := textproto.MIMEHeader{
 			"Content-Type":   {typ},
 			"Content-Length": {strconv.Itoa(length)},
+			"Rendition-Name": {cxn.params.profiles[i].Name},
```

Comment:
Confirming that this'll work with `presets` as well as `profiles` in a webhook response?

Reply:
Yes - if a name isn't specified in the webhook, then we'll create a default name. The client will then have to do a little more work to match up with the rendition, but they should really just set a profile name if they plan to use multipart.

https://github.com/livepeer/go-livepeer/blob/6a485e197c9df6acc0067e55dc80a274fe0cba77/server/mediaserver.go#L205-L211